nautobot/circuits/templates/circuits/circuit.html

```diff
@@ -1,65 +1,14 @@
-{% extends 'base.html' %}
+{% extends 'generic/object_detail.html' %}
 {% load buttons %}
 {% load custom_links %}
 {% load helpers %}
 {% load plugins %}
```

Comment:
Is there any straightforward way to audit whether all of these `load` are still needed in each page? I see that you already removed `load static` from quite a few, but I would have thought that `custom_links` and `plugins` (at least!) should also no longer be needed on specific templates since that's now provided by the generic template?

Reply:
Yeah agreed there. I think it comes down to asserting references to things inside the template that are provided by those modules. It was easy with `load static` since it provides the `static` template tag.

Reply:
I'll go ahead and clean these up. Looks like `buttons`, `custom_links`, `plugins` can be removed from most of them.